recipes/paratext/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cd python/
+$PYTHON setup.py build install --prefix=$PREFIX
```

Comment:
I am not sure you need the build step here. I think that `install` should build and install altogether. Since you are using `setuptools` can you change that to:

`$PYTON setup.py install --single-version-externally-managed --record record.txt`